db/migration/V001__Setup_Tables.sql

```diff
@@ -37,6 +37,7 @@ CREATE TABLE queue
     encrypted_resp text NOT NULL,
     date_sent timestamp without time zone NOT NULL,
     date_received timestamp without time zone NOT NULL,
+    downloaded boolean DEFAULT false,
```

Comment:
For existing schemas, this will break the migration. It would be better to create a new migration file with the change.

Reply:
Wouldn't this also have been the case for [this pull-request](https://github.com/NUMde/compass-numapp-backend/pull/37) ? There, the initial setup of the 'studyparticipant'-table has been modified instead of altering the one created before.

Reply:
You are right. 
From my point of view, it is fine to add changes to existing migration files as long as a project is in development and the db schema is recreated from time to time. But once an application is in production, db changes must be put in new migration files as otherwise the whole concept of versioning you db schema is not working as intended.

Reply:
Fair point.
I'd suggest that from here on we'll only work with new migrations in order to prevent any trouble @mahvaezi.